regexes/device/mobiles.yml

```diff
@@ -4381,7 +4381,7 @@ Smartfren:
 
 # Smartisan
 Smartisan:
-  regex: 'SM(?:70[15]|801|901|919)|YQ60[1357][ /;\)]'
+  regex: 'SM(?:70[15]|801|901|919|105|103)|OD10[35]|YQ60[1357][ /;\)]'
```

Comment:
You forgot to remove the `|105|103`. Otherwise SM103/5 would be detected, or are those valid Smartisan devices?